manifests/redhat/agent6.pp

```diff
@@ -4,7 +4,7 @@
 #
 
 class datadog_agent::redhat::agent6(
-  $baseurl = "https://yum.datadoghq.com/beta/${::architecture}/",
+  $baseurl = "https://yum.datadoghq.com/stable/6/x86_64/",
```

Comment:
This is great, but belongs in a separate PR. I will cherry-pick it to a different branch.